content/en/docs/tasks/traffic-management/istioctl-describe/index.md

```diff
@@ -0,0 +1,241 @@
+No traffic will arrive due to the virtual service destinations.
+
+{{< text plain >}}
+VirtualService: reviews
+   WARNING: No destinations match pod subsets (checked 1 HTTP routes)
+      Route to non-matching subset v1 for (everything)
+{{< /text >}}
+
+Oh no!  I must revert!  I'll delete the bogus Istio configuration:
+
+{{< text bash >}}
+$ kubectl delete -f samples/bookinfo/networking/destination-rule-all-mtls.yaml
+{{< /text >}}
+
+If I refresh the browser at this point the stars do not appear.  Instead I see
+*Error fetching product details!* and *Error fetching product reviews!*  Instead of
+panic, I `describe`:
+
+{{< text bash >}}
+$ istioctl x describe pod $REVIEWS_V2_POD
+{{< /text >}}
+
+{{< text plain >}}
+...
+VirtualService: reviews
+   WARNING: No destinations match pod subsets (checked 1 HTTP routes)
+      Warning: Route to UNKNOWN subset v1.  No DestinationRule.
+{{< /text >}}
+
+At this point I look back and realize I deleted the destination rules, not the virtual service V1 I wished to.  To fix the problem:
+
+{{< text bash >}}
+$ kubectl delete -f samples/bookinfo/networking/virtual-service-all-v1.yaml
+$ kubectl apply -f samples/bookinfo/networking/destination-rule-all-mtls.yaml
+{{< /text >}}
+
+Reloading the browser shows the app has reappeared along with the stars.  `istioctl experimental describe pod $REVIEWS_V2_POD` no longer gives warnings.
+
+# mutual TLS
+
+Let's follow the [Mutual TLS Migration](/docs/tasks/security/mtls-migration/) instructions to enable strict mutual TLS, but targeting ratings:
+
+{{< text bash >}}
+$ kubectl apply -f - <<EOF
+apiVersion: "authentication.istio.io/v1alpha1"
+kind: "Policy"
+metadata:
+  name: "ratings-strict"
+spec:
+  targets:
+  - name: ratings
+  peers:
+  - mtls:
+      mode: STRICT
+EOF
+{{< /text >}}
+
+Now `istioctl x describe pod $RATINGS_POD` reports
+
+{{< text plain >}}
+Pilot reports that pod enforces mTLS and clients speak mTLS
+{{< /text >}}
+
+That's locked down!
+
+If things break when mutual TLS is made `STRICT` it often means that the destination rule didn't match.  For example, if I _destination-rule-all.yaml_ is used instead of `destination-rule-all-mtls.yaml`:
+
+{{< text bash >}}
+$ kubectl apply -f samples/bookinfo/networking/destination-rule-all.yaml # Should have been -mtls
+{{< /text >}}
+
+At this point the browser shows *Ratings service is currently unavailable*.  Why?
+
+{{< text bash >}}
+$ istioctl x describe pod $RATINGS_POD
+{{< /text >}}
+
+The output is the same except the final line which now reads
+
+{{< text plain >}}
+WARNING Pilot predicts TLS Conflict on ratings-v1-f745cf57b-qrxl2 port 9080 (pod enforces mTLS, clients speak HTTP)
+  Check DestinationRule ratings/default and AuthenticationPolicy ratings-strict/default
+{{< /text >}}
+
+Restore correct behavior with `kubectl apply -f samples/bookinfo/networking/destination-rule-all-mtls.yaml`.
+
+## Validation of Istio requirements
+
```

Comment:
nice blog Ed!  I recommend moving this portion earlier.  As user needs to get to mesh first before doing VS/DR.